README.md

```diff
@@ -59,6 +59,7 @@ leto-modelizer
 ├ cypress              ⇨ Contains all the cypress related files
 │ ├ e2e                ⇨ Contains all the e2e tests
 │ ├ videos             ⇨ Contains all videos of each run test (only locally)
+│ └ support            ⇨ Contains all step definitions for e2e tests
```

Comment:
│ └ support            ⇨ Contains all step definitions for e2e tests

TO

│ └ support            ⇨ Contains all supports files for e2e tests
│ │└ step_definitions            ⇨ Contains all step definitions for e2e tests

(something like this)